core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -224,6 +225,23 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
   def listBillingAccounts(): Future[PerRequestMessage] =
     gcsDAO.listBillingAccounts(userInfo) map(RequestComplete(_))
 
+  def getBillingProjectStatus(projectName: RawlsBillingProjectName): Future[PerRequestMessage] = {
+    val statusFuture: Future[Seq[RawlsBillingProjectStatus]] = for {
+      policies <- samDAO.getPoliciesForType(SamResourceTypeNames.billingProject, userInfo)
+      projectDetail <- dataSource.inTransaction { dataAccess => dataAccess.rawlsBillingProjectQuery.load(projectName) }
+    } yield {
+      policies.filter { policy =>
+        policy.resourceId.equals(projectDetail.get.projectName.value)
+      }.flatMap { samResource =>
+        Some(RawlsBillingProjectStatus(RawlsBillingProjectName(samResource.resourceId), projectDetail.get.status))
```

Comment:
Any reason not to reuse the `projectName` parameter here? Is Sam giving us some sort of alternative/more informative name to use? (If so, a comment would be helpful.)